src/Microsoft.DotNet.Build.Tasks/CleanupVSTSAgent.cs

```diff
@@ -414,8 +414,8 @@ private string GetTEMPDirectory()
                     return Environment.GetEnvironmentVariable("TMPDIR");
                 else if (DirExists(Environment.GetEnvironmentVariable("TMP")))
                     return Environment.GetEnvironmentVariable("TMP");
-                else if (DirExists("/home/DDITAdministrator/myagent/_work/_temp"))
-                    return "/home/DDITAdministrator/myagent/_work/_temp";
+                else if (DirExists(Path.Combine(Environment.GetEnvironmentVariable("HOME"), "myagent/_work/_temp")))
```

Comment:
Why not follow the `AddDirToListIfExist` pattern which is used in GetNugetCacheDirectories?